src/components/userComponents/Exhibit/Exhibit.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Image from 'next/image';
+
+/**
+ *
+ * @param root0 passed in
+ * @param root0.title title of exhibit
+ * @param root0.description description of exhibit
+ * @param root0.category category of exhibit
+ * @param root0.image image
+ * @param root0.id id of exhibit
+ * @returns exhibit component
+ */
+export default function Exhibit({
+  title,
+  description,
+  image,
+  id,
+}: {
+  title: string;
+  description: string;
+  image: string;
+  id: string;
+}) {
+  return (
+    <ul key={id}>
```

Comment:
you might have meant a <li> element here (which is a list element) vs a ul which is an unordered list. from my understanding, there is a container around a lot of these components, so li would be appropriate

the normal syntax of an unordered list:

```
<ul>
     <li></li>
     <li></li>
     <li></li>
</ul>
```